cmd/jet/main.go

```diff
@@ -170,6 +178,7 @@ func usage() {
 		"source", "dsn", "host", "port", "user", "password", "dbname", "schema", "params", "sslmode",
 		"path",
 		"ignore-tables", "ignore-views", "ignore-enums",
+		"model-pkg", "table-pkg", "view-pkg", "enum-pkg",
```

Comment:
You'll need to update this names as well.

Reply:
Yeah, my bad.
Pushed the changes already.

Reply:
Great. LGTM. :+1: